sample.py

```diff
@@ -0,0 +1,90 @@
+import os
```

Comment:
did you mean to commit this file too?

Reply:
Yes, this is the request I sent with some modifications to the code I previously committed. I would appreciate it if you could reflect it!